extension/chrome/elements/compose-modules/formatters/signed-msg-mail-formatter.ts

```diff
@@ -40,8 +41,8 @@ export class SignedMsgMailFormatter extends BaseMailFormatter {
       // Removing them here will prevent Gmail from screwing up the signature
       newMsg.plaintext = newMsg.plaintext.split('\n').map(l => l.replace(/\s+$/g, '')).join('\n').trim();
       const signedData = await MsgUtil.sign(signingPrv, newMsg.plaintext);
-      const allContacts = [...newMsg.recipients.to || [], ...newMsg.recipients.cc || [], ...newMsg.recipients.bcc || []];
-      ContactStore.update(undefined, allContacts, { lastUse: Date.now() }).catch(Catch.reportErr);
+      const recipients = Value.arr.unique(Object.values(newMsg.recipients).reduce((a, b) => a.concat(b), []).map(x => x.email));
```

Comment:
Since this line `Value.arr.unique(Object.values(newMsg.recipients).reduce((a, b) => a.concat(b), []).map(x => x.email));` repeats a few times across the files, we could give it a name and make a method to describe what it does. I suppose `getUniqueRecipientEmails` ?

Reply:
You could try `compose-recipients-module.ts` as a public method, then access it in other modules as `this.view.recipientsModule.uniqueRecipientEmails(newMsg.recipients)` or similar.

Reply:
Done

Reply:
Ok, let me fix it...

Reply:
oh no... looks like recipientsModule isn't available from formatters